src/MeetingsAdapter.js

```diff
@@ -0,0 +1,80 @@
+import {throwError} from 'rxjs';
+
+import WebexAdapter from './WebexAdapter';
+
+/**
+ * A meeting object that allows users to have a WebRTC meeting.
+ *
+ * @typedef {Object} Meeting
+ * @property {string}           ID The meeting identifier.
+ * @property {string}           title The title of the meeting.
+ * @property {Date}             startTime The time and date of the start of the meeting.
+ * @property {Date}             endTime The time and date of the end of the meeting.
+ * @property {MediaStreamTrack} localVideo The local video stream track.
+ * @property {MediaStreamTrack} localAudio The local audio stream track.
+ * @property {MediaStream}      localShare The local media share stream.
+ * @property {MediaStreamTrack} remoteVideo The remote video stream track.
+ * @property {MediaStreamTrack} remoteAudio The remote audio stream track.
+ * @property {MediaStream}      remoteShare The remote media share stream.
+ */
+
+/**
+ * The represenation of the desired or current state of an existing meeting.
```

Comment:
```suggestion
 * The representation of the current state of an existing meeting.
```

I would remove the "desired" piece, since that will be done by an action